firebase.go

```diff
@@ -173,6 +173,16 @@ func (fb *Firebase) doRequest(method string, body []byte) ([]byte, error) {
 	}
 
 	resp, err := fb.client.Do(req)
+	if err == nil && resp.StatusCode == http.StatusTemporaryRedirect {
```

Comment:
I'd lean towards moving this to L#190 since we're already checking if `err == nil` there